java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/source/Source.java

```diff
@@ -118,14 +124,16 @@ public static URI getTruffleInternalURI(FileObject fo) {
     
     public static Source getSource(JPDADebugger debugger, long id,
                                    String name,
+                                   String hostMethodName,
                                    String path,
                                    URI uri,
                                    StringReference codeRef) {
-        return getSource(debugger, id, name, path, uri, null, codeRef);
+        return getSource(debugger, id, name, hostMethodName, path, uri, null, codeRef);
     }
 
     public static Source getSource(JPDADebugger debugger, long id,
                                    String name,
+                                   String hostMethodName,
```

Comment:
Class org.netbeans.modules.debugger.jpda.truffle.source.Source
Sigtest reports:
```
  "E1.2 - API type removed" : method 
public final static org.netbeans.modules.debugger.jpda.truffle.source.Source Source.getSource(
  org.netbeans.api.debugger.jpda.JPDADebugger,
  long,
  java.lang.String,java.lang.
  String,java.net.URI,
  com.sun.jdi.StringReference
)
```
consider keeping the old method and passing `null` as the `hostMethodName`.

Reply:
Resolved.